game-core/src/main/java/games/strategy/engine/chat/IChatChannel.java

```diff
@@ -1,29 +1,31 @@
 package games.strategy.engine.chat;
 
 import games.strategy.engine.message.IChannelSubscriber;
+import games.strategy.engine.message.RemoteActionCode;
 import org.triplea.domain.data.UserName;
 import org.triplea.http.client.lobby.chat.ChatParticipant;
 
-/**
- * Chat messages occur on this channel.
- *
- * <p>RMI warning: the ordering of methods cannot be changed, these methods will be invoked by
- * method order number
- */
+/** Chat messages occur on this channel. */
 public interface IChatChannel extends IChannelSubscriber {
   // we get the sender from MessageContext
+  @RemoteActionCode(0)
   void chatOccurred(String message);
 
+  @RemoteActionCode(2)
```

Comment:
Should we re-order the methods so that the numbers would be ascending? I think that might help make it clear there are no gaps and just be nicer when reading the methods.

Reply:
We could, but it would be a tedious process + some classes skip a couple of numbers and if we ever plan to overload a method checkstyle would force us to keep the overloadoverloaded versions together.

We could add this as a "nice to have" task, but this is not something I'd like to address as part of this PR.